src/SocialAuthUserManager.php

```diff
@@ -333,14 +333,10 @@ public function checkIfUserExists($pluginId, $provider_user_id) {
     $social_auth_user = $query->condition('plugin_id', $pluginId)
       ->condition('provider_user_id', $provider_user_id)
       ->execute();
-
-    $user_data = $storage->load(reset($social_auth_user));
-
-    if ($social_auth_user) {
-      // Return User ID.
-      return $user_data->get('user_id')->getValue()[0]['value'];
-    }
-    return FALSE;
+    if (!$social_auth_user)
```

Comment:
You must enclose the block in brackets even if the block contains only one line of code.